packages/components/src/components/page-error/page-error.scss

```diff
@@ -11,6 +11,12 @@
                 justify-content: left;
             }
         }
+        @media screen and (max-width: 1024px) {
+            flex-wrap: wrap;
+        }
+        @media screen and (max-width: 758px) {
+            height: calc(100vh - 224px);
+        }
```

Comment:
do you still need these now?

Reply:
Yes, this properties needed for looking per design on small screen.